hedera-mirror-importer/src/main/resources/db/migration/V1.30.1__hts_support.sql

```diff
@@ -0,0 +1,117 @@
+    (181, 'TOKEN_WAS_DELETED'),
+    (182, 'TOKEN_HAS_NO_SUPPLY_KEY'),
+    (183, 'TOKEN_HAS_NO_WIPE_KEY'),
+    (184, 'INVALID_TOKEN_MINT_AMOUNT'),
+    (185, 'INVALID_TOKEN_BURN_AMOUNT'),
+    (186, 'ACCOUNT_HAS_NO_TOKEN_RELATIONSHIP'),
+    (187, 'CANNOT_WIPE_TOKEN_TREASURY_ACCOUNT'),
+    (188, 'INVALID_KYC_KEY'),
+    (189, 'INVALID_WIPE_KEY'),
+    (190, 'INVALID_FREEZE_KEY'),
+    (191, 'INVALID_SUPPLY_KEY'),
+    (192, 'INVALID_TOKEN_EXPIRY'),
+    (193, 'TOKEN_HAS_EXPIRED'),
+    (194, 'TOKEN_IS_IMMUTABlE');
+
+-- Add token table to hold token properties
+create table if not exists token
+(
+    token_id            entity_id               primary key,
+    create_timestamp    bigint                  not null,
+    divisibility        bigint                  not null,
+    freeze_default      boolean                 not null,
+    freeze_key          bytea,
+    initial_supply      bigint                  not null,
+    kyc_default         boolean                 not null,
+    kyc_key             bytea,
+    modify_timestamp    bigint,
+    supply_key          bytea,
+    symbol              character varying(100)  not null,
+    treasury_account_id entity_id               not null,
+    wipe_key            bytea
+);
+
+--- Add token_account table to capture token-account info such as frozen, kyc and wipe status
+create table if not exists token_account
+(
+    id                  serial              primary key,
+    account_id          entity_id           not null,
+    create_timestamp    bigint              not null,
+    frozen              boolean             not null,
+    kyc                 boolean             not null,
+    modify_timestamp    bigint              not null,
+    token_id            entity_id           not null,
+    wiped               boolean             not null default false
+);
+
+create unique index if not exists token_account__token_account on token_account(token_id, account_id);
+
+create index if not exists token_account__token_account_timestamp
+     on token_account (token_id desc, account_id desc, create_timestamp desc);
+
+--- Add token_balance table to capture token account balances
+create table if not exists token_balance
+(
+    consensus_timestamp bigint              primary key not null,
+    account_id          entity_id           not null,
+    balance             bigint              not null,
+    token_id            entity_id           not null
+);
+
+create index if not exists token_balance__token_account_timestamp
+     on token_balance (token_id desc, account_id desc, consensus_timestamp desc);
+
+--- Add token_transfer
+create table if not exists token_transfer
+(
+    token_id            entity_id,
+    account_id          entity_id,
+    consensus_timestamp bigint,
+    amount              hbar_tinybars
+);
+
+create index if not exists token_transfer__timestamp
+     on token_transfer (token_id desc, account_id desc, consensus_timestamp desc);
```

Comment:
Should be on `consensus_timestamp` only
```suggestion
     on token_transfer (consensus_timestamp desc);
```